packages/notifier/src/publish-kit.js

```diff
@@ -102,6 +111,19 @@ export const subscribeLatest = subscriber => {
 };
 harden(subscribeLatest);
 
+export const PublisherI = M.interface('Publisher', {
+  publish: M.call(M.any()).returns(),
+  finish: M.call(M.any()).returns(),
+  fail: M.call(M.any()).returns(),
+});
```

Comment:
Oh, this is a challenge. My intention is that Published has each of these as optional methods, so that a local observeFoo can feature test for their presence. But I don't think I enabled `M.interface` to express such optionality. Oops.

Reply:
Nevermind. Looking at the types, I see this is the difference between `Publisher` and a `PublishObserver`. Your interface above is exactly right for `Publisher`. Makes sense.